block/olya-iosched.c

```diff
@@ -94,6 +90,8 @@ static int noop_init_queue(struct request_queue *q, struct elevator_type *e)
 	}
 	eq->elevator_data = nd;
 
+	add_queue(q);
```

Comment:
Return value is ignored here. Normally add_queue should return int that in case of error would be returned to the original caller. E.g. in case of memory allocation failure add_queue could return -ENOMEM.